Description: Patch to fix FTBFS due to a modified const in src/callbacks.cc
Forwarded: yes
Origin: Ubuntu,
https://bugs.launchpad.net/ubuntu/+source/freetalk/+bug/443241
Bug-Debian: http://bugs.debian.org/560535
Author: Jon Bernard <bernardj@gmail.com>
--- 3.2-1.orig/src/callbacks.cc	2008-11-15 10:41:22.000000000 +0000
+++ 3.2-1/src/callbacks.cc	2009-10-21 15:50:49.000000000 +0100
@@ -116,13 +116,13 @@ ft_msg_msg_handler (LmMessageHandler *ha
 		    LmMessage *msg, gpointer user_data)
 {
   LmMessageNode *root, *body, *x;
-  const char *from, *msg_str, *type;
-  char *ts = NULL;
+  const char *msg_str, *type;
+  char *from, *ts = NULL;
 
   root = lm_message_get_node (msg);
   body = lm_message_node_get_child (root, "body");
 
-  from = lm_message_node_get_attribute (msg->node, "from");
+  from = (char *) lm_message_node_get_attribute (msg->node, "from");
 
   /* since the file-transfer happens in-band with messages, we can no longer
    * ignore messages with no 'body' */
